docs/releases/5.0.txt

```diff
@@ -233,9 +233,11 @@ CSRF
 Decorators
 ~~~~~~~~~~
 
-* The :func:`~django.views.decorators.cache.cache_control` and
-  :func:`~django.views.decorators.cache.never_cache` decorators now support
-  wrapping asynchronous view functions.
+* The following decorators now support wrapping async view functions:
+  - :func:`~django.views.decorators.cache.cache_control`
+  - :func:`~django.views.decorators.cache.never_cache`
+  - :func:`~django.views.decorators.http.require_safe`
+  - :func:`~django.views.decorators.http.condition`
```

Comment:
We typically put a blank line before a list. I can't recall off the top of my head if this affects rendering, but I suspect it does, so it is probably important to include. 
(You can test by building the docs locally.)

Also, let's alphabetize the list so it's easier to skim:

```suggestion
* The following decorators now support wrapping async view functions:

  - :func:`~django.views.decorators.cache.cache_control`
  - :func:`~django.views.decorators.cache.never_cache`
  - :func:`~django.views.decorators.http.condition`
  - :func:`~django.views.decorators.http.require_safe`
```